src/modules/laydate.js

```diff
@@ -2506,6 +2547,12 @@
       tds.on('click', function(){
         that.choose(lay(this), i);
       });
+
+      if (options.rangeLinked) {
+        tds.on('mouseover', function () {
```

Comment:
```suggestion
        tds.on('mouseenter', function () {
```
1. `mouseenter` 事件不会冒泡，鼠标移入子元素时不会多次触发事件，在使用 cellRender 自定义单元格的场景更合适
2. 鼠标移出 `layui-laydate-content` 元素(table 父元素)时，应该清除临时的高亮效果
3. 建议在 td 点击事件中添加 "changeHover" 的事件，例如在实例上增加一个全局状态，记录是否开启 changeHover 效果，然后在 td 点击事件中切换状态并添加或取消 changeHover 的事件（这一点不重要，可以不做修改）